v3/src/models/codap/add-default-content.ts

```diff
@@ -59,17 +58,10 @@ export function addDefaultComponents() {
       content.insertTileInRow(calculatorTile, row, calcOptions)
     }
 
-    const helloTile = createDefaultTileOfType(kHelloCodapTileType)
-    if (!helloTile) return
-    const helloOptions = isMosaicTileRow(row)
-            ? { splitTileId: calculatorTile.id, direction: "row", percent: 0.75 }
-            : { x: kFullWidth + kWidth25 + kGap, y: 2, width: kWidth75, height: kHalfHeight }
-    content.insertTileInRow(helloTile, row, helloOptions)
-
     const sliderTile = createDefaultTileOfType(kSliderTileType)
     if (sliderTile) {
       const sliderOptions = isMosaicTileRow(row)
-              ? { splitTileId: helloTile.id, direction: "column" }
+              ? { splitTileId: calculatorTile.id, direction: "column" }
               : { x: kFullWidth + kWidth25 + kGap, y: kHalfHeight + kGap, width: kWidth75 }
```

Comment:
```suggestion
              ? { splitTileId: calculatorTile.id, direction: "row" }
              : { x: kFullWidth + kWidth25 + kGap, y: 2, width: kWidth75 }
```
I think you want to split horizontally in the mosaic layout and have the slider be near the top in the free layout.